x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -986,7 +986,7 @@ export const getSavedObjectTypes = (
       name: MESSAGE_SIGNING_KEYS_SAVED_OBJECT_TYPE,
       indexPattern: INGEST_SAVED_OBJECT_INDEX,
       hidden: true,
-      namespaceType: useSpaceAwareness ? 'single' : 'agnostic',
+      namespaceType: 'agnostic',
```

Comment:
Is the `useSpaceAwareness` experimental feature potentially being used anywhere of consequence (by customers, internal deployments, etc)?

Single space ESOs use the object's namespace when constructing AAD. Any instances of Kibana where `useSpaceAwareness` is active, that then upgrade to a version with this change, would cause existing message signing keys and uninstall token objects to be undecryptable.

Reply:
No `useSpaceAwareness` is not used in any long standing deployment, the feature is still in development, we used that feature flag more as an integration branch for now than a real feature flag